transports/janus_rabbitmq.c

```diff
@@ -891,6 +925,8 @@ void *janus_rmq_in_thread(void *data) {
 	timeout.tv_sec = 0;
 	timeout.tv_usec = 20000;
 	amqp_frame_t frame;
+	guint rmq_reconnect_backoff = rmq_reconnect_backoff_initial;
+
 	while(!rmq_client->destroy && !g_atomic_int_get(&stopping)) {
 		amqp_maybe_release_buffers(rmq_client->rmq_conn);
```

Comment:
what happens if connection is getting destroyed (either implicitly or explicitly) and then we re-use the `rmq_conn` ?

Reply:
I'm not sure I understand what you're asking here? Surely we have this covered with the destroy functionality?

Reply:
scratch that, I was worried that the same `rmq_conn` could be reused between retries, but I forgot that `janus_rabbitmq_connect` creates a new connection at every attempt